ExamplePlugins/ExampleNode/Model/CalculateTotalLengthNode.cs

```diff
@@ -0,0 +1,128 @@
+        /// <returns>the documentation</returns>
+        public override IDocumentation CreateDocumentationForTerminal(Terminal terminal)
+        {
+            if (terminal == _inputNamesTerminal)
+            {
+                return new Documentation() { Name = "names" };
+            }
+            else if (terminal == _inputExtraNameTerminal)
+            {
+                return new Documentation() { Name = "extra name" };
+            }
+            return new Documentation() { Name = "length" };
+        }
+
+        /// <summary>
+        /// Our creation method.  This is used to create a new instance either programmatically, from load, and from the palette
+        /// </summary>
+        /// <param name="info">creation information.  This tells us why we are being created (new, load, ...)</param>
+        /// <returns>The newly created node</returns>
+        [XmlParserFactoryMethod(ElementName, ExamplePluginsNamespaceSchema.ParsableNamespaceName)]
+        public static CalculateTotalLengthNode Create(IElementCreateInfo info)
+        {
+            var node = new CalculateTotalLengthNode();
+            node.Init(info);
+            return node;
+        }
+
+        /// <summary>
+        /// Returns the persistence name of this node
+        /// </summary>
+        public override XName XmlElementName
+        {
+            get
+            {
+                return XName.Get(ElementName, ExamplePluginsNamespaceSchema.ParsableNamespaceName);
+            }
+        }
+
+        /// <summary>
+        /// Performs node initialization
+        /// </summary>
+        /// <param name="info">Information about why the node is being created</param>
+        protected override void Init(IElementCreateInfo info)
+        {
+            base.Init(info);
+            Width = StockDiagramGeometries.StandardNodeWidth;
+            Height = StockDiagramGeometries.StandardNodeHeight;
+
+            _inputNamesTerminal = new NodeTerminal(Direction.Input, PFTypes.StringArray1D, "names", TerminalHotspots.Input1);
+            _inputExtraNameTerminal = new NodeTerminal(Direction.Input, PFTypes.String, "extra name", TerminalHotspots.Input2);
+            _outputTerminal = new NodeTerminal(Direction.Output, PFTypes.Int32, "length", TerminalHotspots.CreateOutputTerminalHotspot(TerminalSize.Small, Width, 0));
+            OnComponentInserted(_inputNamesTerminal);
+            OnComponentInserted(_inputExtraNameTerminal);
+            OnComponentInserted(_outputTerminal);
+        }
+
+        /// <summary>
+        /// Returns all of the "Components" of this node.  In this case our components are just our terminals.
+        /// </summary>
+        public override IEnumerable<Element> Components
+        {
+            get
+            {
+                yield return _inputNamesTerminal;
+                yield return _inputExtraNameTerminal;
+                yield return _outputTerminal;
+            }
+        }
+
+        /// <inheritdoc />
```

Comment:
It is best not to use inheritdoc in the examples.

Reply:
Got rid of inheritdoc's in this node and the other example node.